files/zh-cn/learn/javascript/objects/json/index.md

```diff
@@ -133,29 +135,30 @@ superHeroes["members"][1]["powers"][2];
 
 - JSON 是一种纯数据格式，它只包含属性，没有方法。
 - JSON 要求在字符串和属性名称周围使用双引号。单引号无效。
-- 甚至一个错位的逗号或分号就可以导致 JSON 文件出错。你应该小心的检查你想使用的数据 (虽然计算机生成的 JSON 很少出错，只要生成程序正常工作)。你可以通过像 [JSONLint](http://jsonlint.com/) 的应用程序来检验 JSON。
-- JSON 可以将任何标准合法的 JSON 数据格式化保存，不只是数组和对象。比如，一个单一的字符串或者数字可以是合法的 JSON 对象。虽然不是特别有用处……
+- 甚至一个错位的逗号或分号就可以导致 JSON 文件出错。你应该小心的检查你想使用的数据（虽然计算机生成的 JSON 很少出错，只要生成程序正常工作）。你可以通过像 [JSONLint](http://jsonlint.com/) 的应用程序来检验 JSON。
+- JSON 可以将任何标准合法的 JSON 数据格式化保存，不只是数组和对象。比如，一个单一的字符串或者数字可以是合法的 JSON 对象。
 - 与 JavaScript 代码中对象属性可以不加引号不同，JSON 中只有带引号的字符串可以用作属性。
 
-## 主动学习 : 一个 JSON 示例
+## 主动学习：一个 JSON 示例
```

Comment:
Let's align the translation: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#%E6%A0%87%E9%A2%98%E8%A1%A8%E6%A0%BC

```suggestion
## 动手练习：一个 JSON 示例
```